packages/api-openfin/window.js

```diff
@@ -2,4 +2,48 @@ if (!window.ssf) {
   window.ssf = {};
 }
 
-window.ssf.window = (...args) => window.open(...args);
+window.ssf.window = function(url, name, features) {
+  let newWindow;
+  const featureObject = parseFeaturesString(features);
+  const handleError = (error) => console.error('Error creating window: ' + error);
```

Comment:
Yeah that sounds like a good idea. Maybe if we change the features string to an object, we can add this at the same time